src/app/wallet/view.nim

```diff
@@ -36,6 +42,11 @@ QtObject:
     result.currentCollectiblesList = newCollectiblesList()
     result.totalFiatBalance = ""
     result.etherscanLink = ""
+    result.safeLowGasPrice = "0"
+    result.standardGasPrice = "0"
+    result.fastGasPrice = "0"
+    result.fastestGasPrice = "0"
+    result.defaultGasLimit = "22000"
```

Comment:
@iurimatias I set this to 22000, lemme know if this needs change

Reply:
Shouldnt the gas limit be estimated depending on the asset selected?

Reply:
Yes, but the idea was to just go with something for now and then change it later on